### DISCUSSION TITLE
pyXA doesn't support SSH

### DISCUSSION THREAD

**Discussion by stardust-xs** [community/community#64521] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've been using pyXA for some time now and I couldn't understand why it doesn't support authentication using SSH. Any ideas why?

**Comment by xames3** [comment]
Thanks for your question, I'm the author and maintainer for pyXA. Due to remote SSH authentication being susceptible to attacks I've personally not chose to implement this feature. Hope this helps!

**Reply by stardust-xs** [reply]
Aah, gotchu!
Thanks for the prompt response!

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
